types/global.d.ts

```diff
@@ -386,7 +386,7 @@ declare class BoxKey<ValueType> {
   size(): uint64
 }
 
-declare class GlobalStateMap<KeyType, ValueType> {
+declare class GlobalStateMap<KeyType, ValueType, MaxKeys extends number = 1> {
```

Comment:
Rather than having the `MaxKey` as a type arg I think it would be better to have `maxKey` as a constructor object argument, similar to how `key` work for StateKey